tutorials/pages/shimmerevm-hardhat.md

```diff
@@ -0,0 +1,208 @@
+# Deploy a Smart Contract Using Hardhat
+
+:::info
+Hardhat is a powerful development environment for building smart contracts on the Ethereum blockchain. Whether you're new to blockchain development or an experienced developer, Hardhat can help you build better, more reliable smart contracts.
+
+In this tutorial, you will learn how to set up Hardhat and use it to build, test and deploy a simple smart contract on ShimmerEVM.
+:::
```

Comment:
```suggestion
:::info

Hardhat is a powerful development environment for building smart contracts on the Ethereum blockchain. Whether you're new to blockchain development or an experienced developer, Hardhat can help you build better, more reliable smart contracts.

In this tutorial, you will learn how to set up Hardhat and use it to build, test and deploy a simple smart contract on ShimmerEVM.

:::
```

Reply:
Docusaurus recommends new lines here as best practice. There are some edge cases that could break the formatting if you don't use new lines 👍